geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -1126,7 +1126,8 @@ public void txApplyDestroy(Object key, TransactionId txId, TXRmtEvent txEvent,
                   clearOccured = true;
                 }
                 owner.txApplyDestroyPart2(re, re.getKey(), inTokenMode,
-                    clearOccured /* Clear Conflciting with the operation */);
```

Comment:
“Conflicting” typo.  It was here before but since we changed this line may as well fix it.

Reply:
@mcmellawatt sorry! missed your comment.